pkg/model/request_entity.go

```diff
@@ -34,11 +29,10 @@ type RequestEntity interface {
 }
 
 type requestEntity struct {
-	env         simulator.Environment
-	number      int
-	bufferStock RequestsBufferedStock
-	nextBackoff time.Duration
-	attempts    int
+	env          simulator.Environment
+	number       int
+	routingStock RequestsRoutingStock
+	attempts     int
```

Comment:
We don't need `attempts` anymore.  The routing stock will send to an active replica or fail the request.  Just once.

Reply:
Deleted attempts field in https://github.com/pivotal/skenario/pull/67/commits/dfd4d60851eea3b87cc9d7e4dc950b95bc92c495 commit